stock_request/views/stock_request_views.xml

```diff
@@ -50,16 +50,23 @@
         <field name="arch" type="xml">
             <form string="Stock Requests">
                 <header>
+                    <button name="action_submit"
+                            string="Submit" type="object"
+                            states="draft"
+                            class="oe_highlight"
+                            groups="stock_request.group_stock_request_submit"/>
                     <button name="action_confirm"
-                        string="Confirm" type="object"
-                        attrs="{'invisible': [('state', 'not in', ['draft'])]}"/>
-                    <button name="action_cancel" states="draft,open"
-                            type="object" string="Cancel"/>
-                    <button name="action_draft" states="cancel" type="object"
-                            string="Set to Draft"/>
+                            string="Confirm" type="object"
+                            class="oe_highlight"
+                            states="draft,submitted"/>
```

Comment:
@etobella The confirm button is still there to confirm it...

Reply:
Yes, but it gives a Validation Error...

Reply:
@etobella Can you provide the error?

Reply:
![imagen](https://user-images.githubusercontent.com/28590170/58496432-2f3c8780-817a-11e9-8c70-4750c6453c02.png)

As admin I checked the flag on settings for submitting. I created a SR. I was not able to confirm. I submitted it and I tried to confirm it again. But it kept giving the same error again and again

Reply:
If you are in the group to submit, it means someone else not in that group needs to confirm... This is the expected behavior.

Reply:
Internal Users can create stock requests and submit them. The users in the Warehouse User group can then review and confirm.